ui/src/components/core/combobox/combobox.tsx

```diff
@@ -153,18 +157,15 @@ export type ComboboxProps = {
 };
 
 export const Combobox = forwardRef<HTMLInputElement, ComboboxProps>(
-  (
-    {
-      items,
-      selectedValue = { value: '', label: '' },
-      withAutocomplete = false,
-      leftIcon = 'search',
-      onChange,
-      onBlur,
-      error = false,
-    },
-    ref
-  ) => {
+  ({
+    items,
+    selectedValue = { value: '', label: '' },
+    withAutocomplete = false,
+    leftIcon = 'search',
+    onChange,
+    onBlur,
+    error = false,
+  }) => {

```

Comment:
Maybe this is for other PR, but if we are not using the `ref` parameter there is no need to use `forwardRef`.